docs/pages/access-controls/access-request-plugins/opsgenie.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Access Requests with Opsgenie
+description: How to set up Teleport's Opsgenie plugin for privilege elevation approvals.
+---
+
+With Teleport's Opsgenie integration, engineers can access the infrastructure
+they need to resolve alerts quickly, without longstanding admin permissions
+that can become a vector for attacks.
+
+Teleport's Opsgenie integration allows you to treat Teleport Role Access
+Requests as Opsgenie alerts, notify the appropriate on-call team, and
+approve or deny the requests via Teleport. You can also configure the plugin to
+approve Role Access Requests automatically if the user making the request is on
+the on-call team for a service affected by an alert.
+
+This guide will explain how to set up Teleport's Access Request plugin for
+Opsgenie.
+
+(!docs/pages/includes/plugins/diagram.mdx api="Opsgenie" action="Listen for Access Requests\nModify Access Requests" messages="Send notifications\nGet alert status" !)
```

Comment:
If you look in the rendered guide, this diagram is not accurate. It shows the self-hosted plugin architecture. Let's just remove it.
```suggestion
```